verification/challengeresponse.go

```diff
@@ -158,8 +158,8 @@ func (cfg ChallengeResponseConfig) ChallengeResponse(
 	// if requested, explicitly call DELETE on the session resource
 
 	if cfg.DeleteSession {
-		if err = cfg.Client.DeleteResource(uri); err != nil {
-			log.Printf("DELETE %s failed: %v", uri, err)
+		if err2 := cfg.Client.DeleteResource(uri); err2 != nil {
```

Comment:
I presume if DELETE fails, there is an implicit deletion after a timeout ?

Has this part been tested ?

Reply:
you mean tested in véraison services?

Reply:
Yes, I meant, is there a way to test this part ?

Reply:
no, it's a server side behaviour, we can't see it from here.